cmd/common/oci.go

```diff
@@ -50,8 +50,8 @@ import (
 type CommandMode string
 
 const (
-	CommandModeRun    CommandMode = "run"
-	CommandModeAttach CommandMode = "attach"
+	CommandModeRun    CommandMode = "run GADGET"
+	CommandModeAttach CommandMode = "attach GADGET_INSTANCE"
```

Comment:
Usually required params go like `<gadget>`. Maybe use that style here too?

Reply:
Usually optional args go like `[optional arg]` but it seems the required ones are not so established. LGTM, then :)